src/components/FlowDiagram/FlowDiagram.js

```diff
@@ -48,6 +48,13 @@ const FlowDiagram = ({ data, lastDate, prevDate }) => (
             height={28.375}
             isDisabled={data[ADMISSIONS].isNotAvailable || data[POPULATION_PRISON].isNotAvailable}
           />
+          <Arrow
```

Comment:
So we have separate desktop and mobile arrows defined for each transition, each with the same `isDisabled` logic but different width, height, and placement fields.

I worry about the `isDisabled` logic accidentally drifting apart for two arrows in a pair, or about a future engineer accidentally adding only one arrow and not remembering that there needs to be two. So I wonder if it would make more sense to consolidate each pair of arrows into a single one, passing in `isMobile` as an actual boolean, and passing in separate fields for width, height, and placement. Something like this, maybe:

```
<Arrow
  desktopHeight={28.375}
  mobileWidth={2.5}
  mobileHeight={51}
  mobilePlacement="left"
  isDisabled={data[ADMISSIONS].isNotAvailable || data[POPULATION_PRISON].isNotAvailable}
  isMobile={someIsMobileCheck}
/>
```

That muddies up the `Arrow` interface a little bit, but because we've already decided to add an `isMobile` flag to it, the interface is already aware that there is a distinction between desktop and mobile.

Reply:
@jessex great point, I completely agree with you!